packages/components/chip/src/ChipClearButton.styles.tsx

```diff
@@ -55,8 +55,12 @@ export const chipClearButtonWrapperStyles = cva(
   }
 )
 
+// eslint-disable-next-line tailwindcss/no-custom-classname
 export const chipClearButtonStyles = cva(
-  ['rounded-full', 'focus-visible:outline-none focus-visible:u-ring-inset'],
+  [
+    'rounded-full text-[calc(var(--sz-6))] p-[var(--sz-2)] border-sm',
```

Comment:
If I'm not wrong, you can directly consume your CSS variables:
```suggestion
    'rounded-full text-[--sz-6] p-[--sz-2] border-sm',
```

Reply:
thks

Reply:
omg. this doesnt-work 🤯

Reply:
I revert the changes. When applying `text-[--sz-6]` the result css applied is `color: var(--sz-6)` (0.325rem) which does nothing.

Reply:
@acd02 p[--sz-2] [font-size:--sz-8]

Reply:
@acd02 To precise my initial remark: https://tailwindcss.com/docs/adding-custom-styles#using-arbitrary-values. Official documentation says:
> When using a CSS variable as an arbitrary value, wrapping your variable in var(...) isn’t needed — just providing the actual variable name is enough: 
In this particular case it seems more related to the way `text-*` and `--sz-*` work together, IMHO :thinking:

Reply:
@soykje , maybe that can help 👉 https://tailwindcss.com/docs/adding-custom-styles#resolving-ambiguities

Reply:
Well let's leave @andresin87 with all this options to see what's better in his case :stuck_out_tongue: Loving tw simplicity... :innocent: